apps/web/src/connection/index.ts

```diff
@@ -21,6 +21,8 @@ import { Connection, ConnectionType, ProviderInfo } from './types'
 import { getDeprecatedInjection, getIsCoinbaseWallet, getIsInjected, getIsMetaMaskWallet } from './utils'
 import { UniwalletConnect as UniwalletWCV2Connect, WalletConnectV2 } from './WalletConnectV2'
 
+const DEFAULT_CHAIN_ID = ChainId.HEMI_SEPOLIA
```

Comment:
Should we place this in a "global config" so it is easier to change once i.e. we switch to Hemi mainnet?

Reply:
I replaced one extra `ChainId.HEMI_SEPOLIA` that was in this same file below with this DEFAULT_CHAIN_ID so we only have this one line to change (here: https://github.com/hemilabs/interface/pull/3/commits/86e5c778ba600c823948676982198f575c69d51b).
This sets the list of chains on the ChainSelector only, I'd leave this here for now. I suggest we wait till we have HEMI mainnet and see then to which extend we can have one centralized config. Right now, this is too widely spread on many places, including external packages.